plugins/out_cloudwatch_logs/cloudwatch_api.c

```diff
@@ -1458,8 +1435,7 @@ int put_log_events(struct flb_cloudwatch *ctx, struct cw_flush *buf,
             if (c->resp.data == NULL || c->resp.data_len == 0 || strstr(c->resp.data, AMZN_REQUEST_ID_HEADER) == NULL) {
                 /* code was 200, but response is invalid, treat as failure */
                 if (c->resp.data != NULL) {
-                    flb_plg_debug(ctx->ins, "Could not find sequence token in "
-                                  "response: response body is empty: full data: `%.*s`", c->resp.data_len, c->resp.data);
+                    flb_plg_debug(ctx->ins, "Invalid response: full data: `%.*s`", c->resp.data_len, c->resp.data);
```

Comment:
The unit test mock code still references sequence tokens: https://github.com/fluent/fluent-bit/blob/master/plugins/out_cloudwatch_logs/cloudwatch_api.c#L157

Should double check that the unit tests don't use them explicitly either, IIRC they don't.

Reply:
This is still the correct API response for CloudWatch. The code just needs to avoid using the sequence token that is provided by CloudWatch Logs API.

Cloudwatch logs still returns a sequence token even though it does not need to be used.

I tested a request on POSTMAN and here's the response I got from a cw request
Request:
```
{
    "logEvents":[
        {
            "message":"{\"key\":\"\u0000value-abcdefghijklmnopqrstuvwxyz-1234567890\"}",
            "timestamp":{{TIMESTAMP}}
        }
    ],
    "logGroupName":"fluent_onboarding",
    "logStreamName":"sequence_token_deprecationdummy.1"
}
```

Response:
```
{
    "nextSequenceToken": "49637155503606728095532086839945352906426390997192672850"
}
```

Reply:
oh yes you are right, good call